content/events/2023-KCD-Munich

```diff
@@ -0,0 +1,12 @@
+title = "Kubernetes Community Day Munich"
+date = "2023-07-07T00:00:01Z"
+description = "Fermyon engineers Danielle Lancashire and Ryan Levick, along with Tom Gehrke from Liquid Reply, will be giving a \"Hands on with WebAssembly Microservices and Kubernetes\" tutorial at Kubernetes Community Day (KCD) Munich. Don't miss them on Monday July 17 @ 4 PM as they help introduce attendees to WebAssembly."
```

Comment:
Probably wanna include this as Munich time rather than EST 😅

Reply:
I'm not seeing where it says EST? It should be Z(Zulu)/UTC. This line is also for the publishing time for when this file goes live.

Reply:
Possibly the time in the description?

Reply:
Oh I was hoping it was implied it'd be local time relative to where the conf is. Should I add CEST?